.github/dependabot.yml

```
@@ -12,3 +12,15 @@ updates:
   - shyamd
   labels:
   - dependency_updates
+- package-ecosystem: submodules
+  directory: "/"
+  schedule:
+    interval: weekly
```

```suggestion
    interval: weekly
```

I reckon we could make this one daily? We're never going to decline it and it would be good for clients that are continuously deploying this package to remain up to date.

---

Sounds good.